src/sentry/static/sentry/app/views/settings/components/dataPrivacyRules/rulesList.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import TextOverflow from 'app/components/textOverflow';
+import {IconDelete, IconEdit} from 'app/icons';
+import Button from 'app/components/button';
+
+import DataPrivacyRulesForm from './dataPrivacyRulesForm/dataPrivacyRulesForm';
+import {getRuleTypeLabel, getMethodTypeLabel} from './dataPrivacyRulesForm/utils';
+
+type Rule = React.ComponentProps<typeof DataPrivacyRulesForm>['rule'];
+type Props = {
+  rules: Array<Rule>;
+  onShowEditRuleModal?: (id: Rule['id']) => () => void;
+  onDeleteRule?: (id: Rule['id']) => () => void;
+};
+
+const RulesList = React.forwardRef<HTMLUListElement, Props>(function RulesList(
+  {rules, onShowEditRuleModal, onDeleteRule},
+  ref
+) {
+  return (
+    <List ref={ref}>
+      {rules.map(({id, method, type, source}) => {
+        const methodLabel = getMethodTypeLabel(method);
+        const typelabel = getRuleTypeLabel(type);
+        return (
+          <ListItem key={id}>
+            <TextOverflow>
+              {`[${methodLabel.label}] [${typelabel}] ${t('from')} [${source}]`}
+            </TextOverflow>
+            {onShowEditRuleModal && (
+              <Button
+                size="small"
+                onClick={onShowEditRuleModal(id)}
+                icon={<IconEdit />}
+              />
+            )}
+            {onDeleteRule && (
+              <Button size="small" onClick={onDeleteRule(id)} icon={<IconDelete />} />
+            )}
```

Comment:
We should add `label` for these Buttons

Reply:
but why? it' not want we want... do you mean title maybe?